### PR TITLE
doc: add minutes for meeting 2023-02-09

### PR DESCRIPTION
**Pull request by ruyadorno**

### CODE REVIEW

File: doc/meetings/2023-02-09.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* Richard Lau (@richardlau)
+* Juan Arboleda (@juanarbol)
+* Ruy Adorno (@ruyadorno)
+* Beth Griggs (@BethGriggs)
+
+## Agenda
+
+## Announcements
+
+* Richard: Security releases coming up next week
```
Comment (richardlau) [inline doc/meetings/2023-02-09.md:23]:
```suggestion
* Richard: Security releases coming up next week, ref: https://nodejs.org/en/blog/vulnerability/february-2023-security-releases/
```

File: doc/meetings/2023-02-09.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Present
+
+* Release team: @nodejs/release
+* LTS team: @nodejs/lts
+* Rafael Gonzaga: (@RafaelGSS)
+* Richard Lau (@richardlau)
+* Juan Arboleda (@juanarbol)
+* Ruy Adorno (@ruyadorno)
+* Beth Griggs (@BethGriggs)
```
Comment (richardlau) [inline doc/meetings/2023-02-09.md:17]:
```suggestion
* Beth Griggs (@BethGriggs)
* Michaël Zasso (@targos)
```

**Review by richardlau — APPROVED** [review]

File: doc/meetings/2023-02-09.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Links
+
+* **Recording**: https://youtu.be/AARWf6wInDc
+* **GitHub Issue**: https://github.com/nodejs/Release/issues/823
+* **Minutes Google Doc**: https://docs.google.com/document/d/1Sd4yneHzM_vjegIF6Sp7-HPUg_97CgHJ22n5kD661ro
+
+## Present
+
+* Release team: @nodejs/release
+* LTS team: @nodejs/lts
```
Comment (richardlau) [inline doc/meetings/2023-02-09.md:12]:
```suggestion
```

**Review by RafaelGSS — APPROVED** [review]

**Comment by Bladez12192008** [comment]
https://github.com/MetaMask/metamask-extension.gitgh pr

**Comment by Bladez12192008** [comment]
https://github.com/nodejs/Release/pull/826#issuecomment-1425442466

**Pull request merged**